src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,127 +1,96 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
-    /**
-     * Верните текущую дату в виде строки в зависимости от запроса.
-     *
-     * @param datePart Запрос на часть даты или всю дата целиком:
-     *                 - FULL - текущая дата целиком год, месяц, день (число месяца)
-     *                 в формате YYYY-MM-DD, возвращаемое значение по умолчанию;
-     *                 - YEAR - текущий год;
-     *                 - MONTH - название текущего месяца;
-     *                 - DAY - текущий день (число месяца);
-     *                 В любом другом случае бросить DateTimeException
-     **/
+
+    private static final ZoneOffset UKRAINE_TIMEZONE_OFFSET = ZoneOffset.of("+02:00");
+    private static final LocalDate today = LocalDate.now();
+
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return today.toString();
+            case YEAR:
+                return Integer.toString(today.getYear());
+            case MONTH:
+                return Integer.toString(today.getMonthValue());
+            case DAY:
+                return Integer.toString(today.getDayOfMonth());
+            default:
+                throw new DateTimeException("Illegal date part");
+        }
     }
 
-    /**
-     * Верните Optional даты соответствующей дате в массиве.
-     *
-     * @param dateParams Дан массив данных состоящий из 3-х элементов, где:
-     *                   - 1-й элемент массива - год;
-     *                   - 2-й элемент массива - месяц;
-     *                   - 3-й элемент массива - день (число);
-     */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (Exception e) {
```

Comment:
catching `Exception` is too ambiguous

we can catch several particular exceptions in the same catch block, but in case of any trouble we should be able to identify them - for now you can use System.out.println(), later we will learn logging

move return Optional.empty(); to the end of method